app/controllers/application_controller.rb

```diff
@@ -56,6 +59,10 @@ def admin_session_timed_out?
     admin_signed_in? && session[:last_seen_at] < ADMIN_TIMEOUT_LENGTH_IN_MINUTES.minutes.ago
   end
 
+  def redirect_www
```

Comment:
Yes, that will be the case on the final service domain, but we're already sharing the bare domain for the private beta so I think it's ok to stick with that for now as the domain users see and add the `www` redirect for safety.

Reply:
This can probably be done at the routes level with [`constraints`](https://coderwall.com/p/l1uydg/redirect-to-canonical-host-in-rails-4). I'm going to look at doing something similar to address https://trello.com/c/ZRoAp4WJ/529-stop-old-azure-urls-serving-the-site, so will have a PR in shortly

Reply:
Yes, Tekin captures my motivation here.